src/events/pointer.js

```diff
@@ -691,24 +691,29 @@ function pointer(p5, fn){
   fn.pwinMouseY = 0;
 
   /**
-   * A String system variable that contains the value of the last mouse button
-   * pressed.
+   * An object that tracks the current state of mouse buttons, showing which
+   * buttons are pressed at any given moment.
    *
-   * The `mouseButton` variable is either `LEFT`, `RIGHT`, or `CENTER`,
-   * depending on which button was pressed last.
+   * The `mouseButton` object has three properties:
+   * - `left`: A boolean indicating whether the left mouse button is pressed.
+   * - `right`: A boolean indicating whether the right mouse button is pressed.
+   * - `center`: A boolean indicating whether the middle mouse button (scroll wheel button) is pressed.
    *
    * Note: Different browsers may track `mouseButton` differently. See
    * <a href="https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons" target="_blank">MDN</a>
    * for more information.
    *
-   * @property {(LEFT|RIGHT|CENTER)} mouseButton
+   * @property {Object} mouseButton
+   * @property {boolean} mouseButton.left - Whether the left mouse button is pressed.
+   * @property {boolean} mouseButton.right - Whether the right mouse button is pressed.
+   * @property {boolean} mouseButton.center - Whether the middle mouse button is pressed.
    * @readOnly
    *
    * @example
    * <div>
    * <code>
    * function setup() {
-   *   createCanvas(100, 100);
+   *   createCanvas(400, 400);
```

Comment:
This might be a little large on the reference page, would the example still work OK with a smaller canvas like 200x200?

Reply:
This is what 200x200 looks like:
![image](https://github.com/user-attachments/assets/748df7e6-b479-4dd0-8515-bf60ec40d882)
300x300 looks just fine:
![image](https://github.com/user-attachments/assets/fdb98154-96fc-468b-9b99-5ea1bc1eca3d)
We can also print all buttons separately to fit the 200x200 canvas size:
![image](https://github.com/user-attachments/assets/e3b06db0-9f82-462b-8c24-30cf40b32fc8)
```js
p.text(`Left: ${p.mouseButton.left}`, p.width / 2, p.height / 2 - 20);
        p.text(`Right: ${p.mouseButton.right}`, p.width / 2, p.height / 2);
        p.text(`Center: ${p.mouseButton.center}`, p.width / 2, p.height / 2 + 20);```

Reply:
The last one is fairly easy to read, let's go with that one. Thanks!